mitosheet/mitosheet/code_chunks/export_to_file_code_chunk.py

```diff
@@ -10,6 +10,33 @@
 from mitosheet.state import State
 from mitosheet.transpiler.transpile_utils import TAB, column_header_to_transpiled_code
 
+# This is a helper function that generates the code for formatting the excel sheet
+def get_format_code(state: State) -> list:
+    code = []
+    formats = state.df_formats
+    for sheetIndex in range(len(formats)):
+        sheet_name = state.df_names[sheetIndex]
+        format = formats[sheetIndex]
```

Comment:
for sheet_name, format in zip(state.df_names, formats):

Reply:
Oh cool! I'll add that